DeepSpeech.py

```diff
@@ -548,22 +579,83 @@ def train():
             init_op = tfv1.variables_initializer(missing_variables)
             session.run(init_op)
             loaded = True
-
-        tfv1.get_default_graph().finalize()
-
+            
         if not loaded and FLAGS.load in ['auto', 'last']:
+            tf.initialize_all_variables().run()
```

Comment:
I tend to remember this was discouraged / deprecated ? cc @reuben

Reply:
Yes, we don't want to initialize all variables here because it'll mask errors when incompatible checkpoints are used.